bundles/org.openhab.binding.smaenergymeter/src/main/resources/ESH-INF/thing/energyMeter.xml

```diff
@@ -12,6 +12,18 @@
 			<channel id="powerOut" typeId="powerOutType"/>
 			<channel id="energyIn" typeId="energyInType"/>
 			<channel id="energyOut" typeId="energyOutType"/>
+			<channel id="powerInL1" typeId="powerInTypeL1"/>
```

Comment:
It's not needed to create new channel-types, because the channel-types are the same as the orignal onces, except for the title. So it's better to remove all all the newly added channel types and define the channels as follows:
```suggestion
			<channel id="powerInL1" typeId="powerInType">
			    <label>Purchased Power L1</label>
			</channel>
```

Reply:
Thank you for your time. However, regarding the matter at hand I disagree. 

From an electrical perspective the sum and all three individual phases have semantically distinct meanings and are not just anonymous channels with different names.

From the docs "The granularity of channel types should be on its semantic level, i.e. very fine-grained: If a Thing measures two temperature values, one for indoor and one for outdoor, this should be modelled as two different channel types. Overriding labels of a channel type must only be done if the very same functionality is offered multiple times, e.g. having an actuator with 5 relays, which each is a simple "switch", but you want to individually name the channels (1-5)."

Reply:
I stand corrected :smile: (time to reread the documentation)
So instead of sharing it with the sumarized channel-types you can shared them between the L1-L3 types.